src/main/rdf/vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL.ttl

```diff
@@ -13527,6 +13596,16 @@ quantitykind:TemperatureRelatedVolume
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
   rdfs:label "temperature-related volume  "@en-US .
 
+quantitykind:TemperatureVariance_NEON
+  a qudt:QuantityKind ;
+  dcterms:description "Variance for NEON temperature data measured in degrees celcius" ;
+  qudt:applicableUnit unit:DEG_C2 ;
```

Comment:
Don't assert the applicable unit in the quantity kind file. Instead, add to the units file:
```
unit:DEG_C2 qudt:hasQuantityKind quantitykind:TemperatureVariance_NEON ;
```

Reply:
Do I need to rerun the formatting workflow after?

Reply:
I re-ran the Format workflow, and got: 
```
Token does not have repo scope on EDIorg/qudt-public-repo

Source was already formatted, nothing was touched.
```